src/Faker/Provider/fa_IR/Company.php

```diff
@@ -15,6 +15,18 @@ class Company extends \Faker\Provider\Company
         '{{companyField}} {{lastName}}',
     );
 
+      protected static $jobTitleFormat = array(
```

Comment:
Can you provide a datasource?

Reply:
datasource is Wikipedia witch it mentioned in code. I just updated titles from that article

Reply:
then please tell it in a phpDoc comment